terraform/modules/core/core.tf

```diff
@@ -398,14 +398,14 @@ resource "aws_autoscaling_policy" "ecs-autoscaling-down-policy" {
 }
 
 resource "aws_cloudwatch_metric_alarm" "average-reserved-cpu-high" {
-  alarm_name          = "${var.project_name}-${var.environment}-average-reserved-cpu-high"
+  alarm_name          = "${var.project_name}-${var.environment}-cluster-average-reserved-cpu-high"
   comparison_operator = "GreaterThanOrEqualToThreshold"
-  evaluation_periods  = "2"
+  evaluation_periods  = "1"
   metric_name         = "CPUReservation"
   namespace           = "AWS/ECS"
   period              = "60"
   statistic           = "Average"
-  threshold           = "70"
```

Comment:
This was never getting triggered at 70, I tested base resting and loaded usage of the CPU on edge and came up with 40 as a better average threshold.